nixos/default.nix

```
@@ -45,6 +46,11 @@ in {
         <option>users.users.‹name?›.packages</option> option.
       '';
 
+      useGlobalPkgs = mkEnableOption ''
+        using <literal>pkgs</literal> from the NixOS modules in Home Manager.
```

Nothing important but perhaps 
```suggestion
        using the system configuration's <literal>pkgs</literal> argument in Home Manager.
```
or something would be more clear?